install/empty_data.php

```diff
@@ -8622,6 +8622,7 @@ public function getEmptyData(): array
                 'keep_disk' => 1,
                 'keep_certificate' => 1,
                 'clean_certificate' => 1,
+                'disable_locked_field' => 0,
```

Comment:
Should it be simply named `lock_fields`? I'm often not confident with negative parameters in code conditions :)

Reply:
agree with that =)

Reply:
done